lib/chef/formatters/indentable_output_stream.rb

```diff
@@ -17,23 +17,14 @@ def initialize(out, err)
         @semaphore = Mutex.new
       end
 
-      def highline
-        @highline ||= begin
-          require "highline"
-          HighLine.new
+      # pastel.decorate is a lightweight replacement for highline.color
+      def pastel
+        @pastel ||= begin
+          require "pastel"
+          Pastel.new
         end
       end
 
-      # Print text. This will start a new line and indent if necessary
-      # but will not terminate the line (future print and puts statements
-      # will start off where this print left off).
-      #
-      # @param string [String]
-      # @param args [Array<Hash,Symbol>]
-      def color(string, *args)
```

Comment:
This appears to be cruft and not called from anywhere.

Reply:
do any plugins in the community use this?

Reply:
I can't imagine a reasonable reason for a plugin poking around in the internals of IndentableOutputStream. If this was in Core::UI or somewhere I would have left it.

Reply:
👍